README.md

```diff
@@ -1 +1,135 @@
-# math-magicians
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Math Magicians React app <a name="about-project"></a>
+<br>
+
+
+<br>
+"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:<br>
+- Make simple calculations.<br>
+- Read a random math-related quote.<br>
+
+<br>
+<br>
+
+## 🛠 Built With <a name="built-with"></a>
+<br>
+
+
+  <summary>Page was built using React</summary>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+
+<br>
+
+- **Project is created using React**
+- **Easu to reuse components for future projects**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+<br>
+
+- [Click me to view deployed project!](this project will be deployed in near future)
+
```

Comment:
- [ ] [OPTIONAL] I'd suggest you remove the "Live demo" section since the web app hasn't been deployed yet as adding a dummy link makes your project look unprofessional ✔